app/Http/Controllers/BillingController.php

```diff
@@ -82,23 +82,28 @@ public function index(): Factory|View
         
         $services = $this->accountBillingController->getServices(get_user()->account_id);
         $dataServiceId = 0;
-        foreach ($services as $service) {
```

Comment:
Ugly dif,f but all ive done is put the existing code in an if statement that checks if company_id is not null as can be the case in v1 sonar.

Reply:
The 'hide whitespace' options work well here, simplifying it down to the important 5 lines.

<img width="282" alt="hide-whitespace" src="https://github.com/SonarSoftwareInc/customer_portal/assets/68166329/0d643ea7-bdbb-4607-a67a-80403402e441">